dev/dll/version.h

```diff
@@ -0,0 +1,15 @@
+﻿#pragma once
+
+#define STRINGIZE2(s) #s
+#define STRINGIZE(s) STRINGIZE2(s)
```

Comment:
Could you add a comment explaining what this is doing?  I know I've seen this pattern before but I can't for the life of me remember what this double macro thing does.

Reply:
It converts the *value* of a variable into a string. Just one level of macro would result in the *name* of the variable being converted into a string. I'll add a comment.